lib/db/mysql.js

```diff
@@ -553,9 +553,9 @@ module.exports = function (log, error) {
 
   // Select : accounts
   // Fields : uid, email, normalizedEmail, emailVerified, emailCode, kA, wrapWrapKb, verifierVersion, authSalt,
-  //          verifierSetAt, createdAt, locale, lockedAt, profileChangedAt
+  //          verifierSetAt, createdAt, locale, lockedAt, profileChangedAt, keysChangedAt
```

Comment:
Is it fair to ask for some documentation of what `keysChangedAt` actually is? I might have missed it, but new contributors might get confused with this because we already have various `keys` in several places :).

Reply:
Yes it is, but it will be forthcoming in https://github.com/mozilla/fxa-auth-server/issues/2674